README.md

```diff
@@ -26,8 +33,28 @@ or
 
 `poetry run python -m systems`
 
-If you're on VS Code, I also have the docstrings through the [__Python Docstring Generator__ extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) on the Google docstring configuration. (we can definitely change the format if you guys want i just think the google config looks beautiful). If you're on some other editor then rip.
+If you're on VS Code, you should consider using the [__Python Docstring Generator__ extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) on the Google docstring configuration. If you prefer other editors, please make sure your commits follow the current docstring convention.
+
+### Using Docker (beta)
+
+*Natively only supported on Linux and MacOS - use WSL 2 on Windows*
+
+1. Clone this repository `git clone https://github.com/relectric-car-team/systems`
+2. `cd` into the `systems` folder
+3. Run `docker build .` and copy the image name from the console as shown below
+   ```
+   Successfully built <image name>
+   ```
+4. Then run the build `docker build <image name>`
+
+### Using Requirements File (not recommended)
+
+[requirements.txt](requirements.txt)
+
```

Comment:
It might be a nice idea to add a screenshot of the architecture diagram here under an architecture heading

```md
## Architecture and Data Flow Diagram

![Architecure and Data Flow Diagram](docs/images/architecure.png)
```

Or something like that